metricz/metricz.py

```diff
@@ -44,6 +44,7 @@ def __init__(self,
                  url=os.environ.get('OAUTH2_ACCESS_TOKEN_URL', OAUTH2_ACCESS_TOKEN_URL),
                  directory=os.environ.get('CREDENTIALS_DIR', CREDENTIALS_DIR),
                  kairosdb_url=os.environ.get('KAIROSDB_URL', KAIROSDB_URL),
+                 uid='uid',
```

Comment:
this should be `scope`, not `uid`.

Reply:
It's called `token_name` in python-tokens library. So renamed it to `token_name` - https://github.com/zalando-stups/python-tokens/blob/master/tokens/__init__.py#L160